assets/charts/components/azure-arc-onboarding/templates/remove.yaml

```diff
@@ -0,0 +1,30 @@
+apiVersion: batch/v1
+kind: Job
+metadata:
+  annotations:
+    "helm.sh/hook": "pre-delete"
+  name: azure-arc-remove
+spec:
+  template:
+    spec:
+      serviceAccountName: azure-arc-onboarding
+      containers:
+      - image: quay.io/kinvolk/az-cli-with-helm:v0.1
+        name: install-azure-arc
+        args:
+          - remove
+        env:
+          - name: AZURE_APPLICATION_CLIENT_ID
+            value: {{ required "applicationClientID is required" .Values.applicationClientID }}
+          - name: AZURE_APPLICATION_PASSWORD
+            valueFrom:
+              secretKeyRef:
+                name: azure-application-password
+                key: azure-application-password
+          - name: AZURE_TENANT_ID
+            value: {{ required "tenantID is required" .Values.tenantID }}
+          - name: CONNECTED_CLUSTER_NAME
+            value: {{ required "clusterName is required" .Values.clusterName }}
+          - name: AZURE_RESOURCE_GROUP
+            value: {{ required " resourceGroup is required" .Values.resourceGroup }}
```

Comment:
Maybe we can put all those env variables in the template, so we de-duplicate `required` etc?